woocommerce/woocommerce-android
# Issue/4413 minor design fixes

This PR fixes small designs issues raised in [this comment](https://github.com/woocommerce/woocommerce-android/pull/4486#issuecomment-891131684) by @Garance91540.

- updates all occurrences of "in-person payments/In-person payments" with "In-Person Payments"
- Replaces illustration on unsupported country screen
- @Garance91540 I decided to keep the "learn more" section on this screen since it's still imho valid - the docs page contains info about supported countries and our plan to add more countries in the near future.


| Light | Dark |
| --- | --- |
| ![Screenshot_20210804-112858_WooCommerce (Dev)](https://user-images.githubusercontent.com/2261188/128158355-ba6725f8-9c6f-4485-823f-742c87dda96e.jpg) | ![Screenshot_20210804-112843_WooCommerce (Dev)](https://user-images.githubusercontent.com/2261188/128158372-e3144698-10d0-4c43-bd9e-76e0128e637c.jpg) |

To test
- either use a store which has in-person payments enabled and is located in any non-US country or manually update code in CardReaderOnboardingChecker and enable [the CARD_READER feature flag](https://github.com/woocommerce/woocommerce-android/blob/98855f3e8d5da0bae9f1e1eb61a1f68ecf06e644/WooCommerce/src/main/kotlin/com/woocommerce/android/util/FeatureFlag.kt#L22) 
1. Tap on App settings
2. Tap on In-person payments
3. Notice unsupported country 

Update release notes:
